dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/FileUtils.java

```diff
@@ -289,6 +291,29 @@ public static void setFileTo755(File file) throws IOException {
         }
     }
 
+    public static void createDirectoryWith775(@NonNull Path path) throws IOException {
+        if (path.toFile().exists()) {
+            return;
+        }
+        if (OSUtils.isWindows()) {
+            Files.createDirectories(path);
+        } else {
+            Path parent = path.getParent();
+            if (parent != null && !parent.toFile().exists()) {
+                createDirectoryWith755(parent);
```

Comment:
```suggestion
                createDirectoryWith775(parent);
```

Can we unify the method `createDirectoryWith755` ? these two method has the same logic.
We can create a new method createDirectory(@NonNull Path path, Set<PosixFilePermission> permissions);

Reply:
Finished, thanks for your advice. PATL. @ruanwenjun